.github/workflows/integration.yml

```diff
@@ -70,70 +70,81 @@ jobs:
             critest: 1
             userns: 0
             jobs: 1
+            timeout: 10
```

Comment:
Why `10` and not `120` or `60`?

Reply:
critest usually runs about 5minutes, so I gave them 10.